cmd/root.go

```diff
@@ -79,9 +77,13 @@ func getLatestReleaseInfo(info chan<- string) {
 
 func getOcClient() *occlient.Client {
 	client, err := occlient.New()
+	checkError(err, "")
+	return client
+}
+
+func checkError(err error, context string) {
 	if err != nil {
-		fmt.Println(err)
+		fmt.Println(errors.Wrap(err, context))
```

Comment:
Some time back we were discussing that it would be nice to display just the last Error. Instead of displaying the whole chain of wrapped errors.  Could we do it here?
It would be great if we could display a nice error message for the user. And show whole error message chain in the debug log.

Reply:
done